### PR TITLE
Cross entropy benchmarking

### PR DESCRIPTION
**Pull request by XiaoMiQC**

### CODE REVIEW

File: =2.2.3 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Collecting matplotlib
```
Comment (vtomole) [inline =2.2.3]: Accidental commit. Please remove.

Comment (XiaoMiQC) [reply =2.2.3]: Done.

File: cirq/experiments/__init__.py [hunk 2]
```diff
@@ -11,3 +11,8 @@
     two_qubit_randomized_benchmarking,
     two_qubit_state_tomography,
 )
+
+from cirq.experiments.cross_entropy_benchmarking import (
+    cross_entropy_benchmarking,
```
Comment (vtomole) [inline cirq/experiments/__init__.py]: Alphabetical: `build_entangling_layers` should come first.

File: cirq/experiments/cross_entropy_benchmarking.py [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+    @property
+    def data(self) -> Sequence[CrossEntropyPair]:
+        """Returns a sequence of CrossEntropyPairs.
+
+        Each CrossEntropyPair is a NamedTuple that contains a cycle number and
+        the corresponding XEB fidelity.
+        """
+        return self._data
+
+    # coverage: ignore
```
Comment (vtomole) [inline cirq/experiments/cross_entropy_benchmarking.py]: Why the coverage ignore?

File: examples/cross_entropy_benchmarking_example.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# coverage: ignore
```
Comment (vtomole) [inline examples/cross_entropy_benchmarking_example.py]: Why are we ignoring coverage here?

Comment (vtomole) [inline examples/cross_entropy_benchmarking_example.py]: Add more documentation like our other examples. A reference to the paper where this comes from e.t.c. https://github.com/quantumlib/Cirq/blob/master/examples/bcs_mean_field.py is a good example.

**Comment by dabacon** [comment]
@vtomole this was a borked merge that had been previously reviewed. @XiaoMiQC has to move on to other things, so I'm going to merge this and then open a pr to address your comments.

**Pull request merged**